src/util/QdmAttributeHelpers.ts

```diff
@@ -10,6 +10,10 @@ export const PRIMARY_TIMING_ATTRIBUTES = [
   "participationPeriod",
   "authorDatetime",
   "resultDatetime",
+  "activeDatetime",
+  "receivedDatetime",
+  "statusDate",
+  "sentDatetime",
```

Comment:
did we decide to treat these as primary timing attrs?

Reply:
[MAT-6185](https://jira.cms.gov/browse/MAT-6185)

They were listed as a bug but I'm unsure of weather this was formally decided. 
Brandon was going off of the ecqi pdf as documentation
https://ecqi.healthit.gov/sites/default/files/QDM-v5.6-508.pdf


Would you think it would be alright to proceed like this?

Reply:
Probably Stan/Christina question. I'm fine with proceed.